components/console_cmd_ifconfig/console_ifconfig.c

```diff
@@ -0,0 +1,647 @@
+/*
+ * SPDX-FileCopyrightText: 2023 Espressif Systems (Shanghai) CO LTD
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include <stdio.h>
+#include <string.h>
+#include "sdkconfig.h"
+#include "lwip/inet.h"
+#include "lwip/netdb.h"
+#include "lwip/sockets.h"
+#include "esp_netif.h"
+#include "esp_eth.h"
+#include "esp_console.h"
+#include "esp_event.h"
+#include "argtable3/argtable3.h"
+#include "esp_log.h"
+#include "esp_netif_net_stack.h"
+#include "lwip/ip6.h"
+#include "lwip/opt.h"
+#include "ethernet_init.h"
+#if IP_NAPT
+#include "lwip/lwip_napt.h"
+#endif
+
+
+typedef struct netif_op_ netif_op;
```

Comment:
Please remove, this statement has no effect, type is defined on the following line

Reply:
Without this forward declaration 
```esp_err_t (*operation)(netif_op *self, int argc, char *argv[], esp_netif_t *esp_netif);``` in ```struct netif_op``` is throwing an error since netif_op is not declared till that line.